brutils/ibge/uf.py

```diff
@@ -30,3 +30,37 @@ def convert_code_to_uf(code):  # type: (str) -> str | None
         result = CODE_TO_UF(code).name
 
     return result
+
+
+def convert_uf_to_name(uf: str) -> str:
+    """
+    Convert a Brazilian UF code (e.g., 'SP') to its full state name ('São Paulo').
+
+    The lookup is case-insensitive and ignores surrounding whitespace.
+
+    Args:
+        uf (str): Two-letter UF code.
+
+    Returns:
+        str: The full state name.
+
+    Raises:
+        ValueError: If `uf` is not a non-empty string or is not a valid UF code.
+
+    Examples:
+        >>> convert_uf_to_name('SP')
+        'São Paulo'
+        >>> convert_uf_to_name('rj')
+        'Rio de Janeiro'
+    """
+    if not uf or not isinstance(uf, str):
+        raise ValueError("UF must be a non-empty string.")
```

Comment:
Não seria interessante validar o tamanho da string, já que o padrão BR pras UFs é ter 2, como a doc-string já sugere?
```suggestion
    if any([
        not uf,
        not isinstance(uf, str),
        len(uf) != 2
    ]):
        raise ValueError("UF must be a 2 char sized string.")
```